common/scala/src/main/scala/whisk/core/entity/WhiskActivation.scala

```diff
@@ -62,7 +62,8 @@ case class WhiskActivation(
     version: SemVer = SemVer(),
     publish: Boolean = false,
     annotations: Parameters = Parameters(),
-    duration: Option[Long] = None)
+    duration: Option[Long] = None,
+    entityType: String)
```

Comment:
Yea... I figured - I apologize for not having more done before making this PR, but today is my last official day as an intern, and I wanted to get the code up before I go. @mdeuser and I had a discussion about why this wasn't a good idea, but we already had it coded, and I haven't had time to change the implementation.